### PR TITLE
There must be at least one result to show the pagination component. Otherwise the pagination is hidden.

### PR DESCRIPTION
**Pull request by benlam-ignw**
## Description

apps/modernization-ui/src/design-system/Pagination/Pagination.tsx:
        ```<div className={total > 0 ? styles.pagination : styles.hidden}>```

apps/modernization-ui/src/design-system/Pagination/pagination.module.scss:
        ```.hidden {
            display: none;
        }```

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-3428

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

### CODE REVIEW

File: apps/modernization-ui/src/design-system/Pagination/Pagination.spec.tsx [hunk 1]
```diff
@@ -111,4 +111,22 @@ describe('When Pagination renders', () => {
 
         expect(toggle).toBeInTheDocument();
     });
+
+    it('should hide the pagination component when the total results are zero', () => {
+        mockTotal = 0;
+
+        const { getByTestId } = render(<Setup />);
+
+        expect(getByTestId('pagination-container-component')).toBeInTheDocument();
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/design-system/Pagination/Pagination.spec.tsx:120]: nit: Usually `ByTestId` is a [last resort](https://testing-library.com/docs/queries/bytestid) query.  I think you could do the same using the `queryByText` and checking for "Showing ..."

File: apps/modernization-ui/src/design-system/Pagination/Pagination.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const Pagination = () => {
     } = usePage();
 
     return (
-        <div className={styles.pagination}>
+        <div className={total > 0 ? styles.pagination : styles.hidden} data-testid="pagination-container-component">
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/design-system/Pagination/Pagination.tsx:13]: question:  was the styling added to preserve the height when the component is not shown?

**Review by adamloup-enquizit — APPROVED** [review]

File: apps/modernization-ui/src/design-system/Pagination/Pagination.tsx (hunk 2 above)
Comment (benlam-ignw) [reply apps/modernization-ui/src/design-system/Pagination/Pagination.tsx]: Kind of, I had opted for conditional rendering originally but that caused the bottom border to jump when going from no results state to loading state to results page state.

**Review by hclarkEnq — APPROVED** [review]

**Review by adamloup-enquizit — APPROVED** [review]
👍

**Pull request merged**